models/org.go

```diff
@@ -432,12 +432,35 @@ func queryUserOrgIDs(uid int64) *builder.Builder {
 type MinimalOrg = User
 
 // GetUserOrgsList returns one user's all orgs list
-func GetUserOrgsList(uid int64) ([]*MinimalOrg, error) {
+func GetUserOrgsList(doer *User) ([]*MinimalOrg, error) {
 	var orgs = make([]*MinimalOrg, 0, 20)
-	return orgs, x.Select("id, name, full_name, visibility, avatar, avatar_email, use_custom_avatar").
+	sess := x.NewSession()
+	defer sess.Close()
+
+	err := sess.Select("id, name, full_name, visibility, avatar, avatar_email, use_custom_avatar").
 		Table("user").
-		In("id", queryUserOrgIDs(uid)).
+		In("id", queryUserOrgIDs(doer.ID)).
 		Find(&orgs)
+
+	if err != nil {
+		return nil, err
+	}
+
+	for _, org := range orgs {
+		cond := SearchRepositoryCondition(&SearchRepoOptions{
+			Actor:      doer,
+			Private:    true,
+			OwnerID:    org.ID,
+			AllPublic:  false, // Include also all public repositories of users and public organisations
+			AllLimited: false, // Include also all public repositories of limited organisations
+		})
+		org.NumRepos, err = countRepositoryByCondition(x, cond)
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
Unfortunately this is wildly inefficient - imagine you're the kind of person that has 500 organizations.

We should just do this in one query with a couple of joins - likely including the `in` at 442.

Reply:
See the code removed in https://github.com/go-gitea/gitea/pull/14032/files

Reply:
if too many organizations exist, maybe should paging...
Because of my poor sql knowledge, I don't know how to jon the ``count`` result with ``Organizations``. feel free to do it if you like. I have to stop this pull request now. Thanks.